assets/src/components/p4_button/edit.js

```diff
@@ -1,3 +1,10 @@
+/**
+ * This file is copy of core button block edit.js (https://github.com/WordPress/gutenberg/blob/master/packages/block-library/src/button/edit.js), with customize changes.
```

Comment:
Can you update this link to point to the commit of this file you started from? This currently points to that file in the latest commit of the gutenberg repo, and it looks like it is different from the one you based on (has newer changes I believe).